plone/app/locales/locales/de/LC_MESSAGES/widgets.po

```diff
@@ -134,11 +134,11 @@ msgstr "Bundle von dem dies abhängig ist"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Bundles"
-msgstr ""
+msgstr "Bündel"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "CSS/LESS"
-msgstr ""
+msgstr "CSS/LESS"
```

Comment:
if the translated value is the same as the default it should not be translated but stay empty